.npmignore

```diff
@@ -1,9 +1,6 @@
-src
 scripts
 test
 cov.info
 README.md
-example.coffee
```

Comment:
why not include example.js in .npmignore? tbh, not sure why it was ignored in the first place...

Reply:
Good point, I am not sure either, but it's not necessary to have it there.